src/shared/api/server/candles/utils.ts

```diff
@@ -2,22 +2,30 @@ import { OhlcData, UTCTimestamp } from 'lightweight-charts';
 import { DbCandle } from '@/shared/api/server/candles/types.ts';
 import { addDurationWindow, DurationWindow } from '@/shared/utils/duration.ts';
 
-export const dbCandleToOhlc = (c: DbCandle): OhlcData<UTCTimestamp> => {
+export interface CandleWithVolume {
+  ohlc: OhlcData<UTCTimestamp>;
+  volume: number;
+}
+
+export const dbCandleToOhlc = (c: DbCandle): CandleWithVolume => {
   return {
-    close: c.close,
-    high: c.high,
-    low: c.low,
-    open: c.open,
-    time: (c.start_time.getTime() / 1000) as UTCTimestamp,
+    ohlc: {
+      close: c.close,
+      high: c.high,
+      low: c.low,
+      open: c.open,
+      time: (c.start_time.getTime() / 1000) as UTCTimestamp,
+    },
+    volume: c.direct_volume + c.swap_volume,
```

Comment:
question: can we validate with someone this is the correct way to get volume? For example, isn't swap volume the multi-hop route (?), and if so, should that count?

Reply:
Looks like this is correct.